### ISSUE TITLE
jax tutorial 5: cannot run on GPU (due to clash with pytorch RNG)

### ISSUE THREAD

**Issue by murphyk** [phlippe/uvadlc_notebooks#84]
**Tutorial**: -1 (Fill-in number of tutorial)

https://uvadlc-notebooks.readthedocs.io/en/latest/tutorial_notebooks/JAX/tutorial5/Inception_ResNet_DenseNet.html

**Describe the bug**

Same issue as in https://github.com/phlippe/uvadlc_notebooks/issues/83.
However it crops right at the start - merely importing pytorch causes issues.
See screenshot below

Can be fixed by moving
```
# Seeding for random operations
main_rng = random.PRNGKey(42)
```
before importing pytorch.

<img width="1106" alt="Screenshot 2023-03-18 at 6 41 29 PM" src="https://user-images.githubusercontent.com/4632336/226149161-60800079-e2f2-4a4c-97aa-19c829045379.png">

**Comment by murphyk** [comment]
Here is a minimal colab to reproduce the core problem.
https://colab.research.google.com/drive/1gGDAGotSA0aJ9Nd1ilV9_xWBy6uFvVkU?usp=sharing

**Comment by phlippe** [comment]
Closing the issue for being fixed in efbc9de8a08857f1d074d1063c8c3c861475eea4 and discussion continues in #83
